docs/developers/strategies/architecture.md

```diff
@@ -159,3 +159,85 @@ Below are some of the user functions or properties under `OrderTracker` that you
     Returns a dictionary of order IDs that are being cancelled.
 
     Return type: `Dict[str, float]`
+
+
+## Fee Accounting
+
+### OrderCandidate
```

Comment:
On the strategy's use of the `OrderCandidate`, I think we can add something like, "Once the candidate order has been sized by the budget checker, the strategy can examine the sized order to get more information such as `OrderCandidate.collateral_dict` to get a dictionary of the costs associated with the order, or `OrderCandidate.potential_returns` to get an idea of the token and amount of the returns associated with the order."